npmboxxer.js

```diff
@@ -52,7 +52,44 @@
 	};
 
 	var tarExtract = function(source,target,callback) {
-		new targz().extract(source,target,callback);
+		// Unfortunately, the `tar.gz` package at v1.0.2 will sometimes make its
```

Comment:
In addition to the rewrite of the fix, I moved it to where the callback gets made instead of mixing it in with the other unbox logic. Seemed to me that this is a cleaner way to include it.